src/routes/tools/degree-days/_store.js

```diff
@@ -0,0 +1,61 @@
+import { writable, derived } from "svelte/store";
+import climvars from "~/helpers/climate-variables";
+import {
+  CLIMATE_INDICATORS,
+  CLIMATE_VARIABLE,
+  DEFAULT_CLIMATE_INDICATOR,
+  DEFAULT_THRESHOLD_DEGREES,
+  DEFAULT_FREQUENCY_CODE,
+  DEFAULT_SELECTED_MONTHS,
+} from "./_constants";
+
+// List of climvars used in Degree Days Tool
+export const climvarList = climvars
+  .filter(({ id }) => id === CLIMATE_VARIABLE)
+  .map((d) => ({ ...d, title: `Degree Days ${d.label}` }));
+
+// Q: should this even be a store if it's only ever a single value and doesn't change?
+export const climvarStore = (() => {
```

Comment:
I don't think you need a climvar store for this tool.

Reply:
It's definitely overkill and could remove it, but it gets destructured in ExploreData and is referenced various places there. It might be worth keeping just for maintaining consistency between tools.